guides/v2.3/graphql/reference/categories.md

```diff
@@ -3,11 +3,11 @@ group: graphql
 title: category endpoint
 ---
 
-The `category` endpoint allows you to search for a single category definition or the entire category tree.
+The `category` endpoint allows you to search for a single category definition or the entire category tree. To return multiple category levels in a single call, define the response so that it contains up to ten nested `children` options. You cannot return the entire category tree if it contains more than 10 sublevels.
```

Comment:
Just a note, nesting level limit can be re-configured via di.xml by the system integrator.